### DISCUSSION TITLE
A list of examples would be very helpful

### DISCUSSION THREAD

**Discussion by pzdkn** [writer/writer-framework#311] [Ideas]
I think having a list of examples with different usage scenarios would be helpful.

Btw is it possible to supply the source code for the repeater example?

https://www.streamsync.cloud/repeater.html

**Comment by FabienArcellier** [comment]
Hi @pzdkn ,

I am currently work on a generator to have one page documentation per component. It's an idea to include the source code of the component in an expander closed with the link to github. Is it what you have in mind ?

**Comment by pzdkn** [comment]
Sounds good! Maybe plus an example usage of it in the "pigeon-demo" app ?

**Reply by FabienArcellier** [reply]
Here is the look of the new component documentation page.

![image](https://github.com/streamsync-cloud/streamsync/assets/159559/cec4179a-3500-4ee1-aac7-26799ec65a7f)
